### PR TITLE
Fix to Env Var and Config persisting in options.ini in KOLIBRI_HOME

### LINKED ISSUE #9551: Environment variables and configuration flags for Kolibri options are persisted to the `options.ini` file by `generate_empty_options_file`
## Observed behavior
When using either environment variables or configuration flags like `--port` or `--debug` on the first run of a Kolibri device, these will be written to the empty `options.ini` file generated by `generate_empty_options_file` - this means that settings chosen on the first run by ephemeral configuration like environment variables and runtime flags will be persisted for subsequent runs of the server.

## Expected behavior
Ephemeral configuration should not be persisted to disk.

## User-facing consequences
The main place this could be problematic is if someone is using a base Kolibri to generate a `KOLIBRI_HOME` image and they do not intend their configuration flags to be persisted.

Additionally, Kolibri developers might be confused why flags are being persisted when they shouldn't.

## Steps to reproduce
Run kolibri with a new `KOLIBRI_HOME` directory. Run with the `--debug` flag.

See that `options.ini` contains an uncommented line `DEBUG = True`

## Context
Tell us about your environment, including:
 * Kolibri version: 0.15.5
 * Operating system: Ubuntu

### PR DESCRIPTION
**Pull request by thesujai**
## Summary

This Solution Aims to override the default config of environment variables in `_set_from_envvars`
…

## References

Fixes #9551 
…

## Reviewer guidance

…

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by thesujai** [comment]
Hello @rtibbles, please also have a look at 556de590bc08d94c98140ef968c7cd8d61458f2c, this way the `DEBUG` and other env variables that are generated persist(ephemeral configuration), but they are set to `FALSE`

File: kolibri/utils/options.py [hunk 1]
```diff
@@ -827,7 +827,6 @@ def _set_from_envvars(conf):
                                 optname=optname, section=section
                             )
                         )
-                    conf[section][optname] = os.environ[envvar]
```
Comment (rtibbles) [inline kolibri/utils/options.py]: This change breaks the functionality of this function - which is called `_set_from_envvars` - we only want to side step this in the case that we are writing to the options.ini file. We should be using the `using_env_vars` dict returned from this function to help us know which values to ignore.

**Review by rtibbles — CHANGES_REQUESTED** [review]
This breaks existing functionality, even if it does fix the issue.

File: kolibri/utils/options.py (hunk 1 above)
Comment (thesujai) [reply kolibri/utils/options.py]: Okay i didnt noticed that

**Comment by thesujai** [comment]
@rtibbles I made a change directly in the `generate_empty_options_file` so i dont think it will break any functionality, as it is supposed to run only in initialization process, that too only when the options.ini not already exists (https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/utils/sanity_checks.py#L138)

So ig this should not break any functionality and still solve the issue

**Review by rtibbles — APPROVED** [review]
Excellent! This does exactly what we need, with a very small diff. Code looks good, and manual testing checks out.

**Comment by rtibbles** [comment]
Docs build failure is fixed on the release branch, merging.

**Pull request merged**